indexer/services/comlink/src/controllers/api/v4/compliance-v2-controller.ts

```diff
@@ -335,6 +276,161 @@ function generateAddress(pubkeyArray: Uint8Array): string {
   return toBech32('dydx', ripemd160(sha256(pubkeyArray)));
 }
 
+async function validateSignature(
```

Comment:
can you add documentation that if there is no response, then the signature is valid